block/block_cache_test.go

```diff
@@ -9,38 +9,37 @@ import (
 )
 
 func TestBlockCache(t *testing.T) {
-	require := require.New(t)
```

Comment:
Thanks for submitting a PR!

The scope of the issue was actually just for the test helpers. All these changes where we are deleting the creation of `require` or `assert` for the test are unnecessary and should be reverted. 

there are certain test helpers that accept a `require` or an `assert` as an input. Those were what the issue was referring to and should be updated to just accept a `testing.T` so that the helpers have more flexibility.

Reply:
Hey @MSevey, I actually [requested](https://github.com/rollkit/rollkit/pull/1342#pullrequestreview-1734340388) @trinitys7 to change it everywhere in the codebase so we're only using the `assert` and `require` from `strchr/testify` everywhere in our tests instead of using a mix of it and `assert.New` or `require.New` in the codebase. Do you think that is not a cleaner code practice to adopt?

Reply:
Aligning the use make sense from a coding standard, but it should have been done separately to the issue referenced in the PR description.

I don't have a strong preference between use of `require.New` vs `require.Method(t....` but it should be aligned with the team and then we should update our contributors guide around testing. 

So my original comment still stands that we should split of the two changes. 

Change 1 is the issue reference in the PR description.
Change 2 can be the require/assert changes, assuming the rest of the team is aligned.

Reply:
Sounds good, will close this PR.

Apologies @trinitys7 for this [request](https://github.com/rollkit/rollkit/pull/1342#pullrequestreview-1734340388). Please feel free to submit a separate PR just making this change in helpers mentioned [here](https://github.com/rollkit/rollkit/pull/1342#discussion_r1404471242).